package.json

```
@@ -9,8 +9,9 @@
     "node": ">=20.10.0"
   },
   "scripts": {
-    "start": "wrangler pages dev --port 8080",
-    "watch": "tsx build/esbuild-server.ts",
+    "start": "concurrently \"yarn start-wrangler\" \"yarn watch\"",
```

```suggestion
    "start": "run-s start-wrangler watch",
```

We use npm-run-all for exactly situations like these

---

gotta use run-p actually